### PR TITLE
Docs: 10.2 DataViz what's new updates

### PR DESCRIPTION
**Pull request by nmarrs**
Final updates to dataviz whats new

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 1]
```diff
@@ -110,39 +110,47 @@ _Available in public preview in Grafana Open Source and Enterprise_
...
-_Available in public preview in Grafana Open Source and Enterprise_
+_Generally available in Grafana Open Source and Enterprise_
 
 <!-- Nathan Marrs -->
 <!-- Cloud -->
 
-You can now zoom in on the y-axis of your time series visualizations. This is useful when you want to focus on a specific range of values.
+You can now zoom in on the y-axis of your time series and candlestick visualizations. This is useful when you want to focus on a specific range of values. To zoom in on the y-axis, hold the shift key while clicking and dragging on supported visualizations. Double click to reset the zoom.
 
-To learn more, refer to our [Time series documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/time-series/#TODO).
```
Comment (nmarrs) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:126]: investigated and it appears we don't currently have any documentation related to zoom feature 😬

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 2]
```diff
@@ -110,39 +110,47 @@ _Available in public preview in Grafana Open Source and Enterprise_
...
+### Y-axis zoom support in time series and candlestick visualizations
 
-_Available in public preview in Grafana Open Source and Enterprise_
+_Generally available in Grafana Open Source and Enterprise_
 
 <!-- Nathan Marrs -->
 <!-- Cloud -->
 
-You can now zoom in on the y-axis of your time series visualizations. This is useful when you want to focus on a specific range of values.
+You can now zoom in on the y-axis of your time series and candlestick visualizations. This is useful when you want to focus on a specific range of values. To zoom in on the y-axis, hold the shift key while clicking and dragging on supported visualizations. Double click to reset the zoom.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:124]:
```suggestion
You can now zoom in on the y-axis of your time series and candlestick visualizations. This is useful when you want to focus on a specific range of values. To zoom in on the y-axis on supported visualizations, hold the Shift key while clicking and dragging; double-click to reset the zoom.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 3]
```diff
@@ -110,39 +110,47 @@ _Available in public preview in Grafana Open Source and Enterprise_
...
 ### Data visualization quality of life improvements
 
-_Available in public preview in Grafana Open Source and Enterprise_
+_Generally available in Grafana Open Source and Enterprise_
 
 <!-- Nathan Marrs -->
 <!-- Cloud -->
 
-TBD / WIP - some high level thoughts
+We've made a number of smaller improvements to the data visualization experience in Grafana. Notable changes include:
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:135]:
```suggestion
We've made a number of smaller improvements to the data visualization experience in Grafana.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 4]
```diff
@@ -110,39 +110,47 @@ _Available in public preview in Grafana Open Source and Enterprise_
...
+
+#### Geomap marker symbol alignment options
+
+You can now offset Geomap marker symbols from the underlying data point.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:139]:
```suggestion
You can now offset geomap marker symbols from the underlying data point.
```

**Review by imatwawana — APPROVED** [review]
A few small wording things, but otherwise, this looks good!

**Comment by nmarrs** [comment]
Addressed by [this commit](https://github.com/grafana/grafana/pull/75909/commits/900eced1a89771b2315136b3485d44084d0c337c) + a few others on original what's new branch

**Pull request closed without merging**